sessions/react-effects-and-localstorage/journal-app-localstorage/src/components/EntryForm/index.js

```diff
@@ -0,0 +1,33 @@
+import "./EntryForm.css";
+import Button from "../Button";
+
+function EntryForm({ onAddEntry }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    onAddEntry(data);
+    event.target.reset();
+  }
+
+  return (
+    <form className="entry-form" onSubmit={handleSubmit}>
+      <div className="entry-form__title">New Entry</div>
```

Comment:
a11y: Why do we use a `div` here instead of a heading? The `className`tells us about the title, so students will ask this question, I guess.

Reply:
I changed it to be a `h2`